doc/api/fs.md

```diff
@@ -1470,7 +1470,8 @@ returned by this method has a default `highWaterMark` of 64 kb.
 
 `options` can include `start` and `end` values to read a range of bytes from
 the file instead of the entire file. Both `start` and `end` are inclusive and
-start counting at 0. If `fd` is specified and `start` is omitted or `undefined`,
+start counting at 0, and the value of `start` and `end` must be >= 0 and
+<= 2 ** 53 - 1. If `fd` is specified and `start` is omitted or `undefined`,
```

Comment:
Nit: Instead of the magic number `2 ** 53 - 1`, would mentioning [`Number.MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) be better?

Reply:
Sounds reasonable.
How about adding “which is Number.MAX_SAFE_INTEGER” after “2 ** 53 - 1”?